tap_zendesk_chat/http.py

```diff
@@ -39,6 +38,8 @@ def request(self, endpoint, params={}, url=None, url_extra=""):
             raise RateLimitException()
         try:
             response.raise_for_status()
+        except requests.exceptions.HTTPError as e:
+            raise
```

Comment:
added this so that lloyd doesn't detect the exception and classify the tap run as a failure whenever we encounter this error, still raised though so that we can catch it in streams.py

Reply:
<img width="806" alt="Screen Shot 2023-01-12 at 3 42 22 PM" src="https://user-images.githubusercontent.com/29642644/212204429-ea8df6a3-e0c3-48e4-9ac0-8b7b41f32f5e.png">